src/reverse_search.cpp

```diff
@@ -218,6 +218,20 @@ void ReverseSearch::open_reply(QNetworkReply* reply)
 		response.replace("<title>Multi-service image search - Search results</title>",
 				 "<title>WiseTagger Reverse Search Results</title>");
 
+        /* FIXME: another dirty hack: inserting this js function to one-click-copy tags
+         <th onclick="javascript:
+            var whew_tags = document.querySelectorAll('td.image a img')[0].getAttribute('title').split('Tags: ')[1];
+            const el = document.createElement('textarea');
+            el.value = whew_tags;
+            document.getElementById('pages').appendChild(el);
+            el.select();
+            document.execCommand('copy');
+            document.getElementById('pages').removeChild(el);">Best match (click here to copy tags)</th>
+        */
+        response.replace("<th>Best match</th>",

```

Comment:
This can be done without duplication using raw string literals like this:

```c++
		// FIXME: another dirty hack: inserting this js function to one-click-copy tags
		auto onclick = R"~(
<th onclick="javascript:
var whew_tags = document.querySelectorAll('td.image a img')[0].getAttribute('title').split('Tags: ')[1];
const el = document.createElement('textarea');
el.value = whew_tags;
document.getElementById('pages').appendChild(el);
el.select();
document.execCommand('copy');
document.getElementById('pages').removeChild(el);">Best match (click here to copy tags)</th>)~";

		response.replace("<th>Best match</th>", onclick);

```

There's no need to remove line feeds, browsers ignore them anyway.

Reply:
Also the project uses 8-wide tabs for indentation and spaces for additional alignment after indentation.

Reply:
>What source line this corresponds to? Is this line already commited in the .ts file?

"Could not rename %FILENAME%. File may have been renamed or removed by another application, or file with this name already exists in current directory."
Nope, it's been left unchanged, gotta fix that as well but I'm lost as where it is.

>8-wide tabs
Complied.

>This can be done without duplication
Done and also implemented that other idea you had.